win32ss/user/ntuser/ntuser.h

```diff
@@ -65,4 +65,26 @@ InitDisplayDriver(
     IN PWSTR pwszDeviceName,
     IN PWSTR pwszRegKey);
 
+inline VOID APIENTRY PushW32ThreadLock(PVOID pobj, PTL ptl, PVOID pfnFree)
```

Comment:
These new functions need an explanation for their purpose.

Reply:
Deleted these functions.